src/AddTodoForm.js

```diff
@@ -1,13 +1,26 @@
 import React from "react";
 
-function AddTodoForm () {
-    return(
-        <form>
-            <label htmlFor="todoTitle">Title</label>
-            <input id="todoTitle"/>
-            <button type="submit">Add</button>
-        </form>
-    )
+function AddTodoForm(props) {
+  function handleAddTodo(event) {
+    event.preventDefault();
+
+    const todoTitle = event.target.titel.value;
+    props.onAddTodo(todoTitle);
+    console.log(todoTitle);
+    event.target.reset();
+  }
+  return (
+    <form onSubmit={handleAddTodo}>
+      <label htmlFor="todoTitle">Title</label>
+      <input
+        id="todoTitle"
+        name="titel"
```

Comment:
And it will need to be fixed here as well for everything to still work

```suggestion
        name="title"
```